source/testing/acceptance.md

```diff
@@ -8,39 +8,47 @@ ember g acceptance-test login
 This generates this file:
 
 ```tests/acceptance/login-test.js
-import { test } from 'qunit';
-import moduleForAcceptance from 'my-app/tests/helpers/module-for-acceptance';
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
 
-moduleForAcceptance('Acceptance | login');
+module('Acceptance | login', function(hooks) {
+  setupApplicationTest(hooks);
 
-test('visiting /login', function(assert) {
-  visit('/login');
-
-  andThen(function() {
+  test('visiting /login', async function(assert) {
+    await visit('/login');
     assert.equal(currentURL(), '/login');
   });
 });
 ```
 
-`moduleForAcceptance` deals with application setup and teardown. The last few lines, within
-the function `test`, contain an example test.
+`module` allows you to scope your tests: Any test setup that is done inside of this scope, will
+apply to all test cases contained in this module. You can read more about module specific setup and
+teardown in the section [Setup and teardown](Acceptance#setup-and-teardown).
+Scoping your tests with `module` also allows you to execute your tests independently from other tests.
+For example, to only run your tests from your `login` module, run `ember t -m 'Acceptance | login'`.
```

Comment:
It may be better to use the longer form of the various flags/commands in the docs. Its _probably_ much easier to remember `ember test --module` than `ember t -m`...